.prettierrc.json

```diff
@@ -1,5 +1,9 @@
 {
+  "$schema": "http://json.schemastore.org/prettierrc",
```

Comment:
@giotramu this file should be renamed in `.prettierc` so as to remove this `$schema` reference